modules/ob-defining-the-build-specification.adoc

```diff
@@ -0,0 +1,32 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-runs.adoc
+
+:_content-type: CONCEPT
+[id="ob-defining-the-build-specification_{context}"]
+= Defining the build specification
+
+You can embed a complete `spec.buildSpec` field into your `BuildRun` resource to build the image. The following example shows a `BuildRun` CR that configures the `buildSpec` field:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: BuildRun
+metadata:
+  name: standalone-buildrun
+spec:
+  buildSpec:
+    source:
+      url: https://github.com/shipwright-io/sample-go.git
+      contextDir: source-build
+    strategy:
+      kind: ClusterBuildStrategy
+      name: buildpacks-v3
+    output:
+      image: foo/bar:latest
+----
```

Comment:
The whole section from `source` to `output.image` needs to be indented as specified in the previous comment.